packages/node_modules/pouchdb-for-coverage/package.json

```diff
@@ -2,8 +2,5 @@
   "name": "pouchdb-for-coverage",
   "version": "6.1.0-prerelease",
   "main": "./lib/index.js",
-  "private": true,
-  "browser": {
-    "./lib/index.js": "./lib/index-browser.js"
-  }
```

Comment:
pouchdb-for-coverage is always only used in Node mode; this wasn't doing anything